### PR TITLE
OrtIssue fixups

### PR DESCRIPTION
**Pull request by sschuberth**
Please have a look at the individual commit messages for the details.

---
This change is [Reviewable](https://reviewable.io/reviews/heremaps/oss-review-toolkit/1165)

### CODE REVIEW

File: reporter/src/main/kotlin/reporters/StaticHtmlReporter.kt [hunk 1]
```diff
@@ -146,6 +146,11 @@ class StaticHtmlReporter : Reporter() {
           color: #573a08;
         }
 
+        .ort-report-table tr.ort-hint {
+          background: #fffaf3;
+          color: #573a08;
```
Comment (mnonnenmacher) [inline reporter/src/main/kotlin/reporters/StaticHtmlReporter.kt]: I'd propose to take the chance and set better colors, e.g.:

```
background: #f7f5ff;
color: #1c0859;
```

File: model/src/main/kotlin/OrtIssue.kt [hunk 2]
```diff
@@ -65,12 +65,20 @@ class OrtIssueDeserializer : StdDeserializer<OrtIssue>(OrtIssue::class.java) {
             // For backward-compatibility if only an error string is specified.
             OrtIssue(Instant.EPOCH, "", node.textValue())
         } else {
-            OrtIssue(
-                    timestamp = Instant.parse(node.get("timestamp").textValue()),
-                    source = node.get("source").textValue(),
-                    message = node.get("message").textValue(),
-                    severity = node.get("severity")?.let { Severity.valueOf(it.textValue()) } ?: Severity.ERROR
-            )
+            if (node.has("severity")) {
```
Comment (mnonnenmacher) [inline model/src/main/kotlin/OrtIssue.kt]: Nit: double space in commit message between "the" and "default".

Comment (sschuberth) [reply model/src/main/kotlin/OrtIssue.kt]: Done.

File: reporter/src/main/kotlin/reporters/StaticHtmlReporter.kt (hunk 1 above)
Comment (sschuberth) [reply reporter/src/main/kotlin/reporters/StaticHtmlReporter.kt]: Done (in a new commit, though).

**Pull request merged**